SqlAssignments.sql

```diff
@@ -0,0 +1,1846 @@
+
+
+
+
+Que 15  ->  Give the sums of the amounts from the Orders table, grouped by date, eliminating all those dates where the SUM was not at    least 2000 above  the maximum Amount.
+
+Query  ->SELECT odate,SUM(amt) AS total_amt 
+       ->FROM Orders  
+       ->GROUP BY odate;
++------------+-----------+
+| odate      | total_amt |
++------------+-----------+
+| 1990-03-10 |   8944.59 |
+| 1990-04-10 |   1788.98 |
+| 1990-05-10 |      4723 |
+| 1990-06-10 |  11201.83 |
++------------+-----------+
+4 rows in set (0.00 sec)
+
+
+
+**************************************************************************************************************************************
+
+
+
+Que 16 -> Select all orders that had amounts that were greater than at least one of the orders from October 6.
+
+Query   -> SELECT onum,amt,odate 
+        -> FROM Orders 
+        -> WHERE amt > (SELECT MIN(amt)
+        ->              FROM Orders
+        ->              WHERE odate ='1990-06-10');
++------+---------+------------+
+| onum | amt     | odate      |
++------+---------+------------+
+| 3002 |  1900.1 | 1990-03-10 |
+| 3005 | 5160.45 | 1990-03-10 |
+| 3008 |    4723 | 1990-05-10 |
+| 3009 | 1713.23 | 1990-04-10 |
+| 3011 | 9891.88 | 1990-06-10 |
++------+---------+------------+
+5 rows in set (0.00 sec)
+
+**************************************************************************************************************************************
+
+
+
+Que 17 ->  Write a query that uses the EXISTS operator to extract all salespeople who have customers with a rating of 300.
+
+Query  ->   SELECT S.snum,S.sname,S.city 
+       ->   FROM salespeople S 
+       ->   WHERE EXISTS( SELECT C.snum                
+                          FROM Customers C               
+                          WHERE C.snum=S.snum AND  C.rating = 300
+                        );
++------+--------+-----------+
+| snum | sname  | city      |
++------+--------+-----------+
+| 1002 | Serres | SanJose   |
+| 1007 | Rifkin | Barcelona |
++------+--------+-----------+
+2 rows in set (0.01 sec)
+
+
+**************************************************************************************************************************************
+
+
+
+Que 18  ->  Find all customers whose cnum is 1000 above the snum of Serres.
+
+Query   -> SELECT C.CNUM,C.CNAME 
+        -> FROM Customers C,Salespeople S 
+        -> WHERE S.SNUM=C.SNUM and C.CNUM>S.SNUM+1000;
++------+---------+
+| CNUM | CNAME   |
++------+---------+
+| 2003 | Liu     |
+| 2004 | Grass   |
+| 2005 | Clemens |
+| 2007 | Pereira |
++------+---------+
+
+
+
+**************************************************************************************************************************************
+
+
+
+Que 19  -> Give the salespeople’s commissions as percentages instead of decimal numbers.
+
+Query   -> SELECT sname,comm*100 AS COMM 
```

Comment:
I thought a little bit about this query. The output should be in the format as shown below:

+---------+------+
| SNAME   | COMM |
+---------+------+
| Peel    | .12% |
| Serres  | .13% |
| AxelRod | .10% |
| Motika  | .11% |
| Rifkin  | .15% |
| Fran    | .25% |
+---------+------+

Reply:
Okay sir